docassemble/ALToolbox/display_template.py

```diff
@@ -31,16 +46,21 @@ def display_template(
 
         # 2.1 If collapsible, add collapsible elements to the output
         if collapse:
-            return f'<a class="collapsed" data-bs-toggle="collapse" href="#{the_id}" role="button" aria-expanded="false" aria-controls="collapseExample"><span class="pdcaretopen"><i class="fas fa-caret-down"></i></span><span class="pdcaretclosed"><i class="fas fa-caret-right"></i></span> {template.subject_as_html(trim=True)}</a><div class="collapse" id="{the_id}">{text}</div>'
+            return f'<div id="{actual_container_id_tag}" class="{container_classname_plus}"><a class="collapsed" data-bs-toggle="collapse" href="#{the_id}" role="button" aria-expanded="false" aria-controls="collapseExample"><span class="toggle-icon pdcaretopen"><i class="fas fa-caret-down"></i></span><span class="toggle-icon pdcaretclosed"><i class="fas fa-caret-right"></i></span><span class="subject">{template.subject_as_html(trim=True)}</span></a><div class="collapse" id="{the_id}">{text}</div></div>'
 
         # 2.2 If not collapsible, simply return output from copy_button_html()
         else:
-            return text
+            return f"""
+<div id="{actual_container_id_tag}" class="{container_classname_plus}">
+<div class="panel-heading"><h3 class="subject">{template.subject_as_html(trim=True)}</h3></div>
```

Comment:
A thing that was already present, and just being expanded on: if there's not subject, there will be an empty h3. We should instead just not generate a subject element, or maybe make it a div instead.